metadata_backend/api/operators.py

```diff
@@ -1,7 +1,6 @@
 """Operators for handling database-related operations."""
 import re
-import secrets
-import string
+import uuid
```

Comment:
```suggestion
from uuid import uuid4
```

and it will make the syntax simpler